lib/ci/tacos-plan

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 set -euxo pipefail
 export TERRAGRUNT_INCLUDE_MODULE_PREFIX=false
-cmd=(tf-lock "$TF_ROOT_MODULE" -- env --chdir="$TF_ROOT_MODULE" terragrunt-noninteractive run-all plan -out tfplan)
+if [ $GITHUB_PR_DRAFT = "false" ]
+then
```

Comment:
json and posix match in the sense that `true` and `false` are valid posix commands, so

```suggestion
if "$GITHUB_PR_DRAFT"; then
```